gax-java/gax/src/test/java/com/google/api/gax/retrying/AbstractRetryingExecutorTest.java

```diff
@@ -223,14 +223,14 @@ public void testTotalTimeoutExceeded() throws Exception {
 
   @Test
   public void testCancelOuterFutureBeforeStart() throws Exception {
-    FailingCallable callable = new FailingCallable(4, "request", "SUCCESS", tracer);
+    FailingCallable callable = new FailingCallable(0, "request", "SUCCESS", tracer);
```

Comment:
The parameter is `expectedFailuresCount` and it should be `0`. The callable's `call()` function should not be invoked.